pkg/service/repair/plan.go

```diff
@@ -422,31 +319,32 @@ func setIntersection(s1, s2 *strset.Set) *strset.Set {
 
 // tablePlan describes repair schedule and state for table.
 type tablePlan struct {
-	Table string
-	Size  int64
-	// Deleted tables are still being sent to workers,
-	// so that their progress can still be updated in a fake way,
-	// as their jobs are not actually sent to Scylla.
-	Deleted bool
+	Table         string
+	Size          int64
+	RangesCnt     int
+	ReplicaSetCnt int
```

Comment:
Maybe `ReplicationFactor` is the better name here ? Especially that it's just a counter, not the reference to nodes belonging to replica.

Reply:
Actually, it counts the amount of different replica sets, not the replica set size.